.github/workflows/test.yml

```diff
@@ -10,8 +10,23 @@ jobs:
       fail-fast: false
       max-parallel: 5
```

Comment:
Might want to bump max-parallel to `10` or more here since the matrix size has increased significantly.

Reply:
Seems like simply removing the limit is most reasonable.